### ISSUE TITLE
Gravity on the Moon

### ISSUE THREAD

**Issue by StevieBranscum** [micdoodle8/Galacticraft#2562]
I think you know that the gravity on the moon is the same as the overworld, yet i haven't tried it on any of the other planets, so I will try those and get back to you.

**Comment by ThomasAunvik** [comment]
I am using Build 63 and it does not happen to me.

Please post mod list, client log and the most important galacticraft version.

**Comment by radfast** [comment]
@StevieBranscum  which Minecraft version?  1.7.10?

Please post your log: ```fml-client-latest.log```

**Comment by StevieBranscum** [comment]
I am on 1.10.2.

**Comment by ThomasAunvik** [comment]
What build?

**Comment by StevieBranscum** [comment]
I am also using build 61, too.

**Comment by ThomasAunvik** [comment]
Tested it in 1.10.2 build 63. Cannot confirm this. Try updating to the latest version.

**Comment by StevieBranscum** [comment]
I'll try 63 and see if it will work.

**Comment by StevieBranscum** [comment]
And quick question, are there moon dungeons? I searched for a good while, and still haven't found one. Is the generation off, or do I just have bad luck?

**Comment by tabepic** [comment]
I believe it tells you in the logs when and where they generate

**Comment by StevieBranscum** [comment]
Except when I tried to look in the logs, it was a jumble of symbols.

**Comment by tabepic** [comment]
You mean it was corrupted?

**Comment by StevieBranscum** [comment]
Want me to give the log?

**Comment by ThomasAunvik** [comment]
Yes Please

**Comment by StevieBranscum** [comment]
My bad. Didn't see the fml logs. Saw the .gz files.

**Comment by StevieBranscum** [comment]
Can't rn since I'm not on my computer. Later I will.

**Comment by radfast** [comment]
Moon Dungeons are rare and, therefore,hard to find, currently

When I'm looking for one, I fly around **and** watch the console, and there's only ~ 1 every 600 blocks

When you find one it should be fairly visible, especially from the air

I'll talk to @micdoodle8 about ways to help players to locate one (my favourite being "ask a Moon Villager"...)

**Comment by tabepic** [comment]
I get the same issue in 1.10.2 GC Build 63 now. (tested in both creative and survival)

**Comment by ThomasAunvik** [comment]
Currently i am confused what issue we are talking about...

**Comment by tabepic** [comment]
We are talking about how when you go to the Moon, gravity is supposed to be less then that of "Earth".
However, we observe the same amount of gravity on the Moon as in the Overworld

**Comment by ThomasAunvik** [comment]
Not an issue that i am having, so sorry.

**Comment by tabepic** [comment]
I wonder if it's because of some kind of mod that @StevieBranscum and I both have...
@StevieBranscum , could you post a list of the mods you are using?

**Comment by StevieBranscum** [comment]
Can't rn. Will be able to in a bit. Even w/63, it still doesn't work

**Comment by radfast** [comment]
Stevie:  some random questions which could be relevant
* are you playing on a server or single player?
* if it's a server does it have **Sponge**?
* same Galacticraft versions on server and client?
* if your language is not en_US, please temporarily change the client to en_US and see if that makes a difference (it shouldn't, but there was a bug like this 2 years back)

**Comment by StevieBranscum** [comment]
Client, internet access not available except phone. Live in the US, so I speak English. Should be able to get mod list today, in about 30 min.

**Comment by StevieBranscum** [comment]
Also, I do have IC2, but is the incompatibility only for power?

**Comment by StevieBranscum** [comment]
And, @tabepic I can't reach those sites.

**Comment by tabepic** [comment]
You mean ptpb.pw ?

**Comment by StevieBranscum** [comment]
Yes. Chrome can't reach it for some reason

**Comment by tabepic** [comment]
Weird, does it come up with an error? Or just something like 'This website could not be found' ?

**Comment by StevieBranscum** [comment]
Yes, exactly, expect it says the connection was reset

**Comment by tabepic** [comment]
It could be because I'm in Canada and you're in the US

**Comment by StevieBranscum** [comment]
I don't think that would cause anything though.

**Comment by tabepic** [comment]
Yeah, I tried using some kind of online traceroute (from the US), and it said it wasn't a valid website, then I tried using a US proxy and that worked fine.
Weird.

**Comment by StevieBranscum** [comment]
I will put the folder on here, instead of on Pastebin, if that's alright.

**Comment by tabepic** [comment]
I don't know what you mean by 'putting the folder here', but directly posting logs in the comments/issues is looked down upon.

**Comment by StevieBranscum** [comment]
https://drive.google.com/open?id=0B5CWYSQNbRdoQ21QVFZsOEI2dlE
GitHub won't allow anything over 10MB.

**Comment by StevieBranscum** [comment]
Mods folder.

**Comment by tabepic** [comment]
I didn't really mean 'upload the mods', I meant give us a list of the mods, but this will work

**Comment by tabepic** [comment]
Seems like the only one we have in common is CodeChicken

**Comment by StevieBranscum** [comment]
How weird.

**Comment by ThomasAunvik** [comment]
So, i downloaded all your mods you had in the mod folder. And i still dont get any issues about that.

**Comment by tabepic** [comment]
Could it be something in the config folder?
I'm going to test that right away

**Comment by tabepic** [comment]
Nope, wasn't the configs. Gravity is still the same as the Overworld on the Moon

**Comment by StevieBranscum** [comment]
Going back to the dungeon topic, found one. I was just missing it

**Comment by tabepic** [comment]
Alright, tried GC Build 63 and CodeChickenLib, and gravity worked fine. Still testing.

**Comment by radfast** [comment]
> So, i downloaded all your mods you had in the mod folder. And i still dont get any issues about that.

wow! this is dedication

>  I do have IC2, but is the incompatibility only for power?

Galacticraft should be totally compatible with IC2 Experimental, from build 63 onwards.  (Not yet fully tested with IC2 Classic.)

**Comment by tabepic** [comment]
I have determined that for me, it was caused by one of these mods:
```
ActuallyAdditions
AdvancedRocketry
B.A.S.E.
BaseMetals
BiomesOPlenty
Bookshelf
CommonCapabilities
Creative Plus
CyclopsCore
DarkUtilities
DeepResonance
Draconic-Evolution
LibVulpes
OreSpawn
UniDict
valkyrielib
WanionLib
```
I will continue testing to narrow it down to a single mod tomorrow.

**Comment by radfast** [comment]
Thank you for narrowing it down, this really helps.  I would look at **AdvancedRocketry** first, this also has a moon.  Then **OreSpawn**

**Comment by StevieBranscum** [comment]
I'll talk to zmasterfun (the guy who made AdvancedRocketry, since him and I have talked before), if that helps.

**Comment by tabepic** [comment]
You're right, it is AdvancedRocketry

**Comment by StevieBranscum** [comment]
So is that it?

**Comment by radfast** [comment]
Most likely.

Are you in fact on **AdvancedRocketry's Moon** and not on the Galacticraft Moon at all?  (For example, are there villages with Alien Villagers?)

**Comment by StevieBranscum** [comment]
Actually, I am on the Galacticraft moon. Found a Moon dungeon, and there are Alien Villagers

**Comment by StevieBranscum** [comment]
I'll remove AdvancedRocketry and see what happens.

**Comment by StevieBranscum** [comment]
AdvancedRocketry was the issue. Gravity works as it should now.

**Comment by StevieBranscum** [comment]
Also, space station creation seemed disabled, except this same mod uses the dim id 2 for the moon. dim id 2 for Galacticraft is the space station.

**Comment by radfast** [comment]
Galacticraft will automatically use a different id, 3, 4, 5, 6, etc for the space station, if 2 is taken - so don't worry about that

It's very possible that Advanced Rocketry clashes in a few ways.  In Minecraft 1.7.10 they made it so that the two mods integrated properly (they actually required Galacticraft to be loaded before their mod worked).  They're pro coders who make Advanced Rocketry, they know what they're doing so I expect this shouldn't be too hard to resolve.

**Comment by StevieBranscum** [comment]
I think that they stopped when they moved to 1.10.2 because this mod wasn't for 1.10.2 when they updated, but I talked to zmaster today, so I think he'll look into it.
